nautobot_chatops_ansible/tower.py

```diff
@@ -31,7 +30,7 @@ def __init__(
             verify_ssl (bool): Verify SSL connections. Defaults to True.
         """
         if tower_uri:
-            self.uri = tower_uri.rstrip()
+            self.uri = tower_uri.rstrip().rstrip("/")
```

Comment:
Should we also make sure that it has the appropriate `http://` here? Based on a different conversation lately about host/uri/url

Reply:
add this to the latest commit